compiler/generator/instructions_compiler.cpp

```diff
@@ -546,6 +553,10 @@ void InstructionsCompiler::compileMultiSignal(Tree L)
         }
     }
 
+    // these two vars are only used for jax
```

Comment:
I guess this should be wrapped in a `if (gGlobal->gOutputLang == "jax") {...}` right ?